go.mod

```diff
@@ -43,6 +42,11 @@ require (
 
 require golang.org/x/sys v0.0.0-20220408201424-a24fb2fb8a0f // indirect
 
+require (
+	github.com/containers/image v3.0.2+incompatible
+	github.com/containers/image/v5 v5.15.0
```

Comment:
What is the reason of this downgrade?, is it because of this `github.com/containers/image` dependency?